### DISCUSSION TITLE
Sending events from Server to Client

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17962] [Questions] [answered]
Is it possible to send events from server to clients?
If there is, then how?

Or maybe with some modules?

**Comment by wobsoriano** [comment, marked as answer]
Websockets? Check the comments [here](https://github.com/nuxt/framework/discussions/1189)

**Reply by xjccc** [reply]
> Websockets? Check the comments [here](https://github.com/nuxt/framework/discussions/1189)

can u update this link . thank u
